### ISSUE TITLE
2 symlinks in 3-n-h-a-s switched from ../../ to ../shared in #1107 nonprod refactor error during #119

### ISSUE THREAD

**Issue by fmichaelobrien** [terraform-google-modules/terraform-example-foundation#1229]
### TL;DR

# 3-networks-hub-and-spoke
- 2 symlinks modified in error during the non-production to nonproduction rename/refactor - the change is to ../common - which itself is a symlink so the double symlink works.  The other 6 of the 2x4 symlinks are unmodified as ../../
see change in #1107
https://github.com/terraform-google-modules/terraform-example-foundation/pull/1107/files#diff-bcfdfb477dea3f75d9d515aa61ce4f3836b223f11d6e4234122778963fbfef4a

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks-hub-and-spoke/envs/nonproduction/access_context.auto.tfvars
```
-../../access_context.auto.tfvars
+../shared/access_context.auto.tfvars
```
there is a 2nd symlink regression in the same directory in 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/3-networks-hub-and-spoke/envs/nonproduction/common.auto.tfvars
```
-../../common.auto.tfvars
+../shared/common.auto.tfvars
```
I'll raise an issue shortly with the fix after I test a full upstream sync of the TEF.

<img width="1315" alt="Screenshot 2024-05-11 at 19 09 03" src="https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/assets/24765473/494087fc-a5ac-47ff-88ae-3e1c4b362999">

### Expected behavior

Symlinks unmodified

### Observed behavior

Same behavior but now running double symlinks for 2 of 8

### Terraform Configuration

```hcl
N/a cb/car default
```

### Terraform Version

```sh
1.3.10
```

### Additional information

During an upstream sync from TEF master to pbmm master.

I'll put up a patch tonight

**Comment by fmichaelobrien** [comment]
patch

```
fmichaelobrien-macbookpro:terraform-example-foundation fmichaelobrien$ git status
On branch gh1229-fix-symlinks
Your branch is up to date with 'origin/gh1229-fix-symlinks'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   3-networks-hub-and-spoke/envs/nonproduction/access_context.auto.tfvars
	modified:   3-networks-hub-and-spoke/envs/nonproduction/common.auto.tfvars

no changes added to commit (use "git add" and/or "git commit -a")
fmichaelobrien-macbookpro:terraform-example-foundation fmichaelobrien$ git diff
diff --git a/3-networks-hub-and-spoke/envs/nonproduction/access_context.auto.tfvars b/3-networks-hub-and-spoke/envs/nonproduction/access_context.auto.tfvars
index 59c7aa1..b0cccce 120000
--- a/3-networks-hub-and-spoke/envs/nonproduction/access_context.auto.tfvars
+++ b/3-networks-hub-and-spoke/envs/nonproduction/access_context.auto.tfvars
@@ -1 +1 @@
-../shared/access_context.auto.tfvars
\ No newline at end of file
+../../access_context.auto.tfvars
\ No newline at end of file
diff --git a/3-networks-hub-and-spoke/envs/nonproduction/common.auto.tfvars b/3-networks-hub-and-spoke/envs/nonproduction/common.auto.tfvars
index 3316647..39aaa46 120000
--- a/3-networks-hub-and-spoke/envs/nonproduction/common.auto.tfvars
+++ b/3-networks-hub-and-spoke/envs/nonproduction/common.auto.tfvars
@@ -1 +1 @@
-../shared/common.auto.tfvars
\ No newline at end of file
+../../common.auto.tfvars
\ No newline at end of file

```
testing on 387
